pyvista/plotting/charts.py

```diff
@@ -3266,17 +3249,16 @@ def area(self, x, y1, y2=None, color="b", label=""):
         y1 : array_like
             Y coordinates of the points on the first outline of the area to draw.
 
-        y2 : array_like, optional
+        y2 : array_like, default: np.zeros_like(x)
```

Comment:
I still don't like how this is rendered in the documentation, but I don't know how this works internally or how to resolve it. So other than the two suggestions below, this LGTM.

Reply:
Looks like it's hard to fix this, so let's leave it as is.